redskyctl/main.go

```diff
@@ -38,6 +41,9 @@ func init() {
 		panic(err)
 	}
 	rand.Seed(int64(binary.LittleEndian.Uint64(b[:])))
+
+	// Prevent Cobra from changing the command order
+	cobra.EnableCommandSorting = false
```

Comment:
curious why?

Reply:
The alphabetical ordering was meaningless, the current ordering is a little more functional/user oriented. Still not quite what I would like, but there is no easy "command groupings" that I am aware of (I think Kube had to build that themselves).